scene/gui/text_edit.cpp

```diff
@@ -3675,7 +3675,6 @@ void TextEdit::_push_current_op() {
 		return; // do nothing
 
 	if(next_operation_is_complex) {
-		current_op.chain_forward=true;
```

Comment:
@StraToN Hm, Ctrl+F "chain_forward" does return matches here: https://github.com/godotengine/godot/blob/9d754247d93883804de94425278a3d1a50f2be6e/scene/gui/text_edit.cpp
And it wouldn't be crashing if it wasn't used ;)